### ISSUE TITLE
complete example

### ISSUE THREAD

**Issue by luto65** [drbh/power-message#2]
It would be great if you could provide a complete example.
Furthermore, would be even better if one could use your code also from non-Mac devices (e.g. raspberry).
Thanks

**Comment by njerschow** [comment]
Luto65, check out this API for accessing from any server: [sendblue.co](sendblue.co)

**Comment by jgordonpds** [comment]
yes it would be nice to see an example of receiving messages as well. Whether it be through a web hook or something to that affect
